examples/three_layer_constant_fluxes.jl

```diff
@@ -125,6 +130,12 @@ function parse_command_line_arguments()
             help = "Whether or not to pick the simulation up from latest checkpoint"
             default = false
             arg_type = Bool
+
+        "--name"
```

Comment:
What are we planning to use this name for? Should we pick one naming scheme (either parameter name-value pairs, or an English `name`, rather than both?)